2021-12-16 18:54:42,753 INFO    MainThread:34121 [wandb_setup.py:_flush():69] setting env: {'project': 'InvertibleTargetPropagation', 'entity': 'tatsukichishibuya', 'sweep_param_path': '/Users/tatsukichi/InvertibleTargetPropagation/code/wandb/sweep-vzzgi050/config-rwn4dxwe.yaml', 'sweep_id': 'vzzgi050', 'run_id': 'rwn4dxwe', 'root_dir': '/Users/tatsukichi/InvertibleTargetPropagation/code'}
2021-12-16 18:54:42,753 INFO    MainThread:34121 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-16 18:54:42,753 INFO    MainThread:34121 [wandb_init.py:_log_setup():336] Logging user logs to /Users/tatsukichi/InvertibleTargetPropagation/code/wandb/run-20211216_185442-rwn4dxwe/logs/debug.log
2021-12-16 18:54:42,754 INFO    MainThread:34121 [wandb_init.py:_log_setup():337] Logging internal logs to /Users/tatsukichi/InvertibleTargetPropagation/code/wandb/run-20211216_185442-rwn4dxwe/logs/debug-internal.log
2021-12-16 18:54:42,754 INFO    MainThread:34121 [wandb_init.py:init():369] calling init triggers
2021-12-16 18:54:42,754 INFO    MainThread:34121 [wandb_init.py:init():376] wandb.init called with sweep_config: {'batch_size': 117, 'hid_dim': 428, 'learning_rate': 0.09236493325471032}
config: {'problem': 'classification-70000', 'algorithm': 'BP', 'epochs': 100, 'seed': 39, 'depth': 4, 'in_dim': 784, 'out_dim': 10, 'activation function': 'leakyrelu'}
2021-12-16 18:54:42,754 INFO    MainThread:34121 [wandb_init.py:init():418] starting backend
2021-12-16 18:54:42,754 INFO    MainThread:34121 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-16 18:54:42,762 INFO    MainThread:34121 [backend.py:ensure_launched():132] starting backend process...
2021-12-16 18:54:42,768 INFO    MainThread:34121 [backend.py:ensure_launched():137] started backend process with pid: 34131
2021-12-16 18:54:42,770 INFO    MainThread:34121 [wandb_init.py:init():423] backend started and connected
2021-12-16 18:54:42,774 INFO    MainThread:34121 [wandb_run.py:_config_callback():787] config_cb None None {'batch_size': 117, 'hid_dim': 428, 'learning_rate': 0.09236493325471032}
2021-12-16 18:54:42,774 INFO    MainThread:34121 [wandb_init.py:init():465] updated telemetry
2021-12-16 18:54:42,775 INFO    MainThread:34121 [wandb_init.py:init():484] communicating current version
2021-12-16 18:54:43,723 INFO    MainThread:34121 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-16 18:54:43,723 INFO    MainThread:34121 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-12-16 18:54:43,982 INFO    MainThread:34121 [wandb_init.py:init():522] starting run threads in backend
2021-12-16 18:54:48,992 INFO    MainThread:34121 [wandb_run.py:_console_start():1565] atexit reg
2021-12-16 18:54:48,993 INFO    MainThread:34121 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-12-16 18:54:48,993 INFO    MainThread:34121 [wandb_run.py:_redirect():1417] Redirecting console.
2021-12-16 18:54:48,994 INFO    MainThread:34121 [wandb_run.py:_redirect():1479] Redirects installed.
2021-12-16 18:54:48,995 INFO    MainThread:34121 [wandb_init.py:init():546] run started, returning control to user process
2021-12-16 18:55:10,860 INFO    MainThread:34121 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 255
2021-12-16 18:55:10,864 INFO    MainThread:34121 [wandb_run.py:_restore():1507] restore
2021-12-16 18:55:12,876 INFO    MainThread:34121 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 871
  total_bytes: 871
}

2021-12-16 18:55:13,185 INFO    MainThread:34121 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 871
  total_bytes: 871
}

2021-12-16 18:55:14,082 INFO    MainThread:34121 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 871
  total_bytes: 22854
}

2021-12-16 18:55:14,475 INFO    MainThread:34121 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22854
  total_bytes: 22854
}

2021-12-16 18:55:14,582 INFO    MainThread:34121 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22854
  total_bytes: 22854
}

2021-12-16 18:55:14,686 INFO    MainThread:34121 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22854
  total_bytes: 22854
}

2021-12-16 18:55:14,792 INFO    MainThread:34121 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22854
  total_bytes: 22854
}

2021-12-16 18:55:14,897 INFO    MainThread:34121 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22854
  total_bytes: 22854
}

2021-12-16 18:55:15,003 INFO    MainThread:34121 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22854
  total_bytes: 22854
}

2021-12-16 18:55:15,104 INFO    MainThread:34121 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22854
  total_bytes: 22854
}

2021-12-16 18:55:16,381 INFO    MainThread:34121 [wandb_run.py:_show_files():1879] logging synced files
